### DISCUSSION TITLE
get started using dexie-cloud-addon

### DISCUSSION THREAD

**Discussion by pwsimon** [dexie/Dexie.js#1826] [Q&A] [answered]
i try to build a simple website. (not a complex app using Webpack)
so i start using: [Hello World](https://dexie.org/docs/Tutorial/Hello-World).

How to extend this with DexieCloud

    <script src="https://unpkg.com/dexie@4.0.1-beta.1/dist/dexie.js"></script>
    <script src="https://unpkg.com/dexie-cloud-addon@4.0.1-beta.52/dist/modern/dexie-cloud-addon.js"></script>

but i run into trouble ...
did i need a CommonJS Module or what?

**Comment by pwsimon** [comment, marked as answer]
i found!

    <script src="https://unpkg.com/dexie@4.0.1-beta.1/dist/dexie.js"></script>
    <script src="https://unpkg.com/rxjs@7.8.1/dist/bundles/rxjs.umd.js"></script>
    <script src="https://unpkg.com/dexie-cloud-addon@4.0.1-beta.52/dist/modern/dexie-cloud-addon.js"></script>
    <script>
        var db = new Dexie("MySyncedDB", {addons: [DexieCloud.dexieCloud]});
    <script>

**Reply by dfahlander** [reply]
Correct except that the dexie-cloud-addon should be the UMD version:

```html
<script src="https://unpkg.com/dexie-cloud-addon/dist/umd/dexie-cloud-addon.js"></script>
```

**Comment by dfahlander** [comment]
Here's an update answer to this question:

#### HTML

```html
<script src="https://unpkg.com/dexie"></script>
<script src="https://unpkg.com/rxjs@7/dist/bundles/rxjs.umd.js"></script>
<script src="https://unpkg.com/dexie-cloud-addon/dist/umd/dexie-cloud-addon.js"></script>
```

#### JS
```js
const db = new Dexie('MySyncedDB', { addons: [DexieCloud.dexieCloud] });
```
